src/docker.py

```diff
@@ -45,7 +48,11 @@ def extract_resources_from_docker(self):
         elif self.chain_name == 'equilibrium':
             self.__extract_from_docker('equilab/eq-para', '/usr/local/bin/paranode', '/etc/chainspec.json')
         elif self.chain_name in ['pendulum', 'amplitude']:
-            self.__extract_from_docker('pendulumchain/pendulum-collator', '/usr/local/bin/pendulum-collator')
+            try:
+                self.__extract_from_docker('pendulumchain/pendulum-collator', '/usr/local/bin/pendulum-collator')
+            except ValueError as e:
+                logger.warning("Could not find pendulum-collator, trying amplitude-collator. Error: %s", e)
+                self.__extract_from_docker('pendulumchain/pendulum-collator', '/usr/local/bin/amplitude-collator')
```

Comment:
Actually it couldn't, since both chains use the same binary from the same Docker image and as such the binary name has changed for both of them.

A bit weird that they made this change but it is what I saw in the latest image. And also why I want this `try` in there - in case they change back :sweat_smile: